luis-update.js

```diff
@@ -3,9 +3,8 @@ const la = require('./luis-add');
 
 const update = (myObj2,exampleId) =>{
         return ld.delete(exampleId)
-            .then((responses) => {
-                la.add(myObj2);
-                console.log(responses);
+            .then(() => {
+                return la.add(myObj2);
             })
             .catch((err) =>{
```

Comment:
When you return a promise, you don't need to do catch here, otherwise, you will miss the error outside this function.